sdks/python/test-suites/portable/common.gradle

```diff
@@ -188,6 +212,7 @@ project.task("postCommitPy${pythonVersionSuffix}IT") {
   dependsOn = [
           'setupVirtualenv',
           'installGcpTest',
+          'portableLocalRunnerJuliaSetWithSetupPy',
```

Comment:
I suggest moving this to https://github.com/apache/beam/blob/0f04de72d759a5d0d2f161fa27c3028d37c35c34/sdks/python/test-suites/portable/common.gradle#L183